lib/util/cache/global/redis.ts

```diff
@@ -0,0 +1,77 @@
+/* istanbul ignore file */
+import { IHandyRedis, createHandyClient } from 'handy-redis';
+import { DateTime } from 'luxon';
+import { logger } from '../../../logger';
+
+let client: IHandyRedis | undefined;
+
+function getKey(namespace: string, key: string): string {
+  return `${namespace}-${key}`;
+}
+
+export function end(): void {
+  try {
+    client?.redis?.end(true); // TODO: Why is this not supported by client directly?
```

Comment:
@viceice I wonder if I'm missing something here. Renovate wouldn't exit without the redis client being cleaned up, so I can't understand why it is not a first-class function in handy-redis.

Reply:
strange, `client.quit()` is there